onnxruntime/contrib_ops/cuda/fused_conv.cc

```diff
@@ -94,6 +97,24 @@
       ORT_RETURN_IF_ERROR(onnxruntime::cuda::SliceOutUnwantedOutputSection(
           this->Stream(), Base::s_.y_data, Base::s_.y_dims_with_adjusted_pads, Base::s_.Y->MutableDataRaw(),
           Base::s_.y_dims.GetDims(), Base::s_.slice_starts, Base::s_.slice_ends, Base::s_.slice_axes, Base::s_.element_size));
+
+      if (has_b || has_z) {  // if b or z has to be added after slice
+
+        onnxruntime::cuda::CudnnTensor sliced_y_tensor;
+        ORT_RETURN_IF_ERROR(sliced_y_tensor.Set(Base::s_.y_dims.GetDims(), onnxruntime::cuda::CudnnTensor::GetDataType<CudaT>()));
+
+        if (has_b) {
+          has_b = cudnnStatus_t::CUDNN_STATUS_SUCCESS != cudnnAddTensor(Base::CudnnHandle(), &alpha, Base::s_.b_tensor, Base::s_.b_data,
+                                                                        &alpha, sliced_y_tensor, Base::s_.Y->MutableDataRaw());
+        }
+        if (has_z) {
+          has_z = cudnnStatus_t::CUDNN_STATUS_SUCCESS != cudnnAddTensor(Base::CudnnHandle(), &alpha, Base::s_.z_tensor, Base::s_.z_data,
+                                                                        &alpha, sliced_y_tensor, Base::s_.Y->MutableDataRaw());
+        }
+      }
+    }
+    if (has_b || has_z) {
```

Comment:
>f (has_b || has_z) { [](http://example.com/codeflow?start=5&length=20)

has_b and has_z are used for error reporting and has dual purpose. I think we need to introduce other vars for it.

Reply:
removed "has_b" and "has_z" from error reporting.